### PR TITLE
Add Line endpoints, tests, routes, and seed data

### PR DESCRIPTION
**Pull request by thomas-jung**
Creating a haiku allows user to create a line. Updating a haiku allows user to update the first line. Added index to haiku_id in Lines table. 100% test coverage now.

### CODE REVIEW

File: app/controllers/haikus_controller.rb [hunk 1]
```diff
@@ -13,20 +13,18 @@ def show
   def create
     haiku = Haiku.new(haiku_params)
     if haiku.save
-      render json: haiku
-      head 201
+      render json: haiku.lines
```
Comment (charliemcelfresh) [inline app/controllers/haikus_controller.rb]: Do we need a status 201 here ? I think by default the status code is going to be 200

File: app/controllers/lines_controller.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
+class LinesController < ApplicationController
+  def index
+    lines = Line.eager_load(:haiku).where("haiku_id = ?", params[:haiku_id])
+    render json: lines, each_serializer: LineSerializer
+  end
+
+  def create
+    line = Line.new(line_params)
+    if line.save
+      render json: line
```
Comment (charliemcelfresh) [inline app/controllers/lines_controller.rb]: will need to update this test too

**Comment by charliemcelfresh** [comment]
@thomas-jung looks good -- I'm suggesting two changes, when those are changed, let's merge this

File: app/controllers/haikus_controller.rb (hunk 1 above)
Comment (thomas-jung) [inline app/controllers/haikus_controller.rb]: I'm not sure if I'm understanding correctly. I don't think we need the 201 there, so I removed it. The status would be 200 now and json will show the haiku lines for the haiku created.

**Comment by charliemcelfresh** [comment]
:hamburger:

**Pull request merged**